app/lib/activitypub/activity/create.rb

```diff
@@ -20,8 +20,11 @@ def process_status
     ApplicationRecord.transaction do
       @status = Status.create!(status_params)
 
-      process_tags(@status)
-      process_attachments(@status)
+      begin
+        process_tags(@status)
+        process_attachments(@status)
+      rescue ActiveModel::ValidationError
```

Comment:
Wait, I am against this. Which validation errors do you want to suppress? You could for example use `save` instead of `save!` and `create` instead of `create!` etc to get the same result without a blank error suppression!